ui/webui/src/components/localization/InstallationLanguage.jsx

```diff
@@ -195,14 +201,20 @@ LanguageSelector.contextType = AddressContext;
 
 export const InstallationLanguage = ({ idPrefix, setIsFormValid, onAddErrorNotification }) => {
     return (
-        <Form>
-            <FormGroup label={_("Select the language you would like to use. This language will also be selected for your installed system.")}>
-                <LanguageSelector
-                  idPrefix={idPrefix}
-                  setIsFormValid={setIsFormValid}
-                  onAddErrorNotification={onAddErrorNotification}
-                />
-            </FormGroup>
-        </Form>
+        <>
+            <Text component={TextVariants.p}>{_(
+                "Select the language you would like to use. This language will also be selected" +
+                " for your installed system.")}
+            </Text>
+            <Form>
+                <FormGroup>
```

Comment:
Can you set the `<Text>` directly to the `<FormGroup label=...` instead, or does that break the styles?

Reply:
That's a completely different style.

Reply:
Makes sense, thanks.